daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -514,12 +517,30 @@ class CLIConf(arguments: Array[String]) extends scallop.ScallopConf(arguments) {
     requireSubcommand()
   }
 
+  // Encode or decode EXI Subcommand Options
+  object exi extends scallop.Subcommand("exi") {
+    banner("""|Usage: daffodil exi [-d] [-s <schema>] [-o <output>] [infile]
+              |
+              |Encode/decode an XML file with EXI. If a schema is specified, it will use schema aware encoding/decoding.
+              |
+              |EncodeEXI Options:""".stripMargin)
+
+    descr("Encode an XML file with EXI")
+    helpWidth(width)
+
+    val output = opt[String](argName = "file", descr = "Output file to write the encoded/decoded file to.")
+    val schema = opt[URI]("schema", argName = "file", descr = "DFDL schema to use for schema aware encoding/decoding.")(fileResourceURIConverter)
+    val infile = trailArg[String](required = false, descr = "Input XML file to encode. If not specified, or a value of -, reads from stdin.")
```

Comment:
Can we move infile to the end of the options (i.e. after decode) like we do with the other subcommands? Makes it more clear that it's a trailing argument.